test/Azure.Functions.Cli.Tests/E2E/Helpers/ProcessHelper.cs

```diff
@@ -76,6 +78,37 @@ public static void TryKillProcessForPort(string port)
             }
         }
 
+        public static int GetAvailablePort(int minPort = 7000, int maxPort = 8000)
```

Comment:
I saw a sample on stackoverflow that showed a way to get a random port by asking for a socket (or was it a TCP listener?) with a port of `0`, which lets the OS assign you a random port, then connecting to that socket to see what its port was.

Not sure if you are interested in that

Reply:
Ohh that's cool, I will check it out